app/pages/project/classify.jsx

```diff
@@ -82,7 +86,11 @@ export class ProjectClassifyPage extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    const { classification, upcomingSubjects, workflow } = this.props;
+    const { actions, classification, translations, upcomingSubjects, workflow } = this.props;
+
+    if (translations !== prevProps.translations) {
```

Comment:
`translations` is a nested object, so I'm not sure that you can use direct comparison here to detect changes.

Reply:
lodash has a deep equality function: https://lodash.com/docs/4.17.11#isEqual

Reply:
In other components, I've listened for locale changes in order to detect when the language changes.
https://github.com/zooniverse/Panoptes-Front-End/blob/eb1a1556d22405adea855fbac1bd6c58d9ec2fac/app/pages/project/workflow-selection.jsx#L39-L41

Reply:
Also note that you could dispatch an `updateClassification` action from within the translations ducks file, whenever a new workflow translation loads. Interventions does this in order to update the classifier store from Sugar messages:
https://github.com/zooniverse/Panoptes-Front-End/blob/eb1a1556d22405adea855fbac1bd6c58d9ec2fac/app/redux/ducks/interventions.js#L3